lib/src/facts/aix/memory_resolver.cc

```diff
@@ -33,10 +34,14 @@ namespace facter { namespace facts { namespace aix {
                 // swapqry the things that have an attribute we
                 // expect, but ignore any errno values that look like
                 // "this just wasn't a good device to query"
+                // ENXIO: No such device address.
                 if (errno != ENODEV &&
                     errno != ENOENT &&
-                    errno != ENOTBLK) {
+                    errno != ENOTBLK &&
+                    errno != ENXIO) {
                     throw system_error(errno, system_category());
```

Comment:
I think it'd be good to add the failing device to this exception while we're in here making changes. `system_error`'s constructor can [take a third argument](https://en.cppreference.com/w/cpp/error/system_error/system_error) to add additional context. Probably just `system_error(errno, system_category(), device)` is fine.

We probably use the two-argument form all over the place, but we can fix those up as we encounter them.

Reply:
Updated.